jobs/JREGIONAL_PLOT_ALLVARS

```diff
@@ -52,12 +52,12 @@ sections=(
   user
   nco
   workflow
-  task_plot_allvars
-  task_run_fcst
+  task_plot_allvars.envars
```

Comment:
I set a default in the `task_plot_allvars` section that pointed to FCST_LEN_HRS so that the task_run_fcst section is no longer needed to supply a bash default.